src/components/form-controls/radio-field.tsx

```diff
@@ -0,0 +1,18 @@
+import { InputFieldProps } from '@/models';
+import parse from "html-react-parser";
+
+
+export function RadioField({ name, label, id, ...rest }: InputFieldProps) {
+
+    return (<>
```

Comment:
code format is not triggered 

please re-format the code for the whole code base